app/views/home/faq.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1>FAQ</h1>
+
+<h2>General</h2>
+
+
+<strong>What is diversitytickets.org?</strong>
```

Comment:
`<strong>` is not a block element. It would be cool if you could wrap a separate `<p>` around it or put it within the existing one :)